### ISSUE TITLE
Package request: Ripcord

### ISSUE THREAD

**Issue by Kehorson** [void-linux/void-packages#19576]
**Name:** Ripcord
**Description:** Ripcord is a desktop chat client for group-centric services like Slack and Discord. It provides a traditional compact desktop interface designed for power users. It's not built on top of web browser technology: it has a small resource footprint, responds quickly to input, and gets out of your way.

[Website](https://cancel.fm/ripcord/)
[Shareware Redistribution Rules for Ripcord](https://cancel.fm/ripcord/shareware-redistribution/)

**Comment by zdykstra** [comment]
Does upstream provide a release in any other format than AppImage ?

**Comment by kyunal** [comment]
> Does upstream provide a release in any other format than AppImage ?

No, but according to their redistribution terms it's possible to unpack the AppImage for redistribution via a package manager. [Reference](https://cancel.fm/ripcord/shareware-redistribution/)

**Comment by alferguet** [comment]
> 
> 
> > Does upstream provide a release in any other format than AppImage ?
> 
> No, but according to their redistribution terms it's possible to unpack the AppImage for redistribution via a package manager. [Reference](https://cancel.fm/ripcord/shareware-redistribution/)

One of the requirements for the package to be included in the repos is that it must be compiled before being used so don't think method is allowed.

**Comment by kyunal** [comment]
Unless the developer changes his distribution model this won't be possible since no source code is provided. I guess this can be closed.

**Comment by rc-05** [comment]
This package could be put inside the **non-free** repository since this software is released as a binary only application with a shareware (thus non-free) license.

**Comment by stnby** [comment]
Close this already what are we gonna package next? Microsoft Office?

**Comment by darkralts** [comment]
what does microsoft office have to do with ripcord?

**Comment by rayfadh** [comment]
The software last update was 2021, thus it seems dead.
